include/lsst/afw/image/ExposureInfo.h

```diff
@@ -190,6 +191,16 @@ class ExposureInfo {
     /// Set the exposure's visit info
     void setVisitInfo(std::shared_ptr<image::VisitInfo const> const visitInfo) { _visitInfo = visitInfo; }
 
+    /// Does this exposure have a transmission curve?
+    bool hasTransmissionCurve() const { return static_cast<bool>(_transmissionCurve); }
```

Comment:
I actually think all of these `hasX` methods are probably unnecessary, since the caller can just call `getX` in a boolean context, but I didn't want to break from the de facto convention that had been established in this file, and I definitely didn't want to remove all of the existing `hasX` methods.